core/src/main/java/feign/BaseBuilder.java

```diff
@@ -269,5 +281,19 @@ List<Field> getFieldsToEnrich() {
         .collect(Collectors.toList());
   }
 
+  protected ResponseInterceptor.Chain executionChain() {
```

Comment:
can we consider a more descriptive name?  How about `interceptors` or `responseInterceptorChain?`